sdk/go/common/resource/plugin/plugin.go

```diff
@@ -183,6 +184,15 @@ func newPlugin(ctx *Context, pwd, bin, prefix string, kind workspace.PluginKind,
 		logging.V(9).Infof("Launching plugin '%v' from '%v' with args: %v", prefix, bin, argstr)
 	}
 
+	// Create a root span for the operation
+	opts := []opentracing.StartSpanOption{}
+	if ctx != nil && ctx.tracingSpan != nil {
+		opts = append(opts, opentracing.ChildOf(ctx.tracingSpan.Context()))
+	}
+	opts = append(opts, opentracing.Tag{Key: "pulumi-decorator", Value: prefix + ":" + bin})
```

Comment:
"Pulumi-decorator" is an odd tag key? Why not just tags "prefix" (or plugin because that's what the prefix is supposed to be) and "bin"?

Reply:
We use `pulumi-decorator` on a lot of spans as a tag for a variety of different things. I think we do this to make it easier to convert to a tabular form with https://github.com/pulumi/pulumi-trace-tool. I agree that it's kind of strange. I do think the tabular form is kind of useful so perhaps we could add individual tags as well as an aggregated "pulumi-decorator" tag?

Reply:
Probably, but I guess that can be another thing when we move to Otel. I hadn't seen the Pulumi-decorator thing before, fine to leave as is for now.